enzy_htp/core/env_manager.py

```diff
@@ -224,9 +224,15 @@ def run_command(self,
 
     def __getattr__(self, key: str) -> str:
         """Allows accession into acquired executables."""
+        if key.startswith('__') and key.endswith('__'):
```

Comment:
so is this to prevent the use of dunder methods being used? what is the purpose of this? was there a specific problem you encountered?

Reply:
Yes I have it in the commit msg but probably should also leave a comment in the code.
This is critical for pickle to work. https://stackoverflow.com/questions/50888391/pickle-of-object-with-getattr-method-in-python-returns-typeerror-object-no